zdp/zdp_handlers.cpp

```diff
@@ -160,6 +160,7 @@ void DeRestPluginPrivate::handleDeviceAnnceIndication(const deCONZ::ApsDataIndic
         if (si->address().ext() == ext)
         {
             si->rx();
+            si->setValue(RAttrLastAnnounced, i->lastRx().toUTC());
```

Comment:
Overlooked that little sweety twice... Needs to be `si->lastRx().toUTC()`

Reply:
Thanks, still haven't seen a _Device Announcement_ for any `/sensors` resource myself.